libtesseract/CMakeLists.txt

```diff
@@ -1,7 +1,9 @@
 cmake_minimum_required(VERSION 2.8.3)
 project(libtesseract)
 
-find_package(catkin REQUIRED)
+find_package(catkin REQUIRED COMPONENTS
+	libleptonica
```

Comment:
@ipa-rmb shouldn't be `libleptonica-dev` https://github.com/ros/rosdistro/blob/09d3b8fba89403c173d55ed0f48863e18eda6f60/rosdep/base.yaml#L1851 ?